WebApp/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "test/**/*.ts"],
```

Comment:
Is this fix needed?

Reply:
`tsconfig.json` used by VSCode for code completion.
No need for build ts code, so I add `tsconfig.build.json` for ts build process. (change npm scripts)